### PR TITLE
infinispan and file device registry tests and move it to smoke tests

### PR DESCRIPTION
**Pull request by famartinrh**

### CODE REVIEW

**Comment by famartinrh** [comment]
@enmasse-ci run tests profile=smoke-iot

File: systemtests/src/test/java/io/enmasse/systemtest/iot/registry/DeviceRegistryTestBase.java [hunk 1]
```diff
@@ -4,43 +4,75 @@
...
+import io.vertx.core.json.JsonArray;
+import io.vertx.core.json.JsonObject;
+
 @Tag(sharedIot)
-class DeviceRegistryTest extends IoTTestBaseWithShared {
+@Tag(smoke)
+public abstract class DeviceRegistryTestBase extends IoTTestBase {
 
-    private Logger log = CustomLogger.getLogger();
+    protected Logger log = CustomLogger.getLogger();
```
Comment (kornys) [inline systemtests/src/test/java/io/enmasse/systemtest/iot/registry/DeviceRegistryTestBase.java]: I don't understand why it is protected?

Comment (famartinrh) [reply systemtests/src/test/java/io/enmasse/systemtest/iot/registry/DeviceRegistryTestBase.java]: my fault I must had change it and have forget about it

Comment (kornys) [reply systemtests/src/test/java/io/enmasse/systemtest/iot/registry/DeviceRegistryTestBase.java]: ok :)

**Comment by kornys** [comment]
@enmasse-ci run tests profile=shared-iot

**Comment by kornys** [comment]
@enmasse-ci run tests profile=smoke-iot

**Pull request merged**